backend/tests/unit/workflows/test_catalogue.py

```diff
@@ -15,13 +16,13 @@
 @pytest.mark.parametrize("workflow", [pytest.param(workflow, id=workflow.name) for workflow in workflows])
 def test_workflow_definition(workflow: WorkflowDefinition) -> None:
     """Validate that we can import the function for each workflow."""
-    workflow.validate_workflow()
+    load_flow_function(module_path=workflow.function, flow_name=workflow.function)
 
 
 @pytest.mark.parametrize("workflow", [pytest.param(workflow, id=workflow.name) for workflow in workflows])
 def test_workflow_definition_matches(workflow: WorkflowDefinition) -> None:
     """Validate that the name of the workflow matches the name of the flow"""
-    flow = workflow.get_function()
+    flow = load_flow_function(module_path=workflow.module, flow_name=workflow.function)
```

Comment:
Not sure how this is better to do it this way ...

Reply:
Initial motivation was that workflow.get_function is only used for testing purposes so I wanted to remove it to avoid confusion when looking at related code. However workflow.get_function seems to be used for automations in non-test code so we might actually let it